include/ktopt_interface/ktopt_planning_context.hpp

```diff
@@ -24,6 +25,10 @@ using drake::planning::trajectory_optimization::KinematicTrajectoryOptimization;
 using drake::solvers::Solve;
 using drake::systems::Diagram;
 using drake::systems::Context;
+using drake::multibody::PackageMap;
+using Eigen::VectorXd;
```

Comment:
Nit: I'd avoid using `using` excessively. For the drake classes it makes sense but I'd avoid using it for the Eigen classes and the joints. Personally, I think that makes the code easier to comprehend

Reply:
So just use the full namespace?

Reply:
Yes